public/apps/account/account-nav-button.tsx

```diff
@@ -62,7 +62,23 @@ export function AccountNavButton(props: {
           }}
           handleSwitchAndClose={() => {
             setModal(null);
-            window.location.reload();
+            // the below portion is to clear URLs starting with 'lastUrl'
+            // when switching tenants, the last URLs will be from the old tenancy therefore we need to remove these from localstorage.
+            const lastUrls = [];
+            for (let i = 0; i < sessionStorage.length; i++) {
+              const key = sessionStorage.key(i);
```

Comment:
`sessionStorage` has not been declared in the file and it may not be fetched. You should declare the variable in the file.

Reply:
In this case, sessionStorage is a property on the global window object. so `sessionStorage === window.sessionStorage`. I've updated it to `window.sessionStorage` for readability.